dockerfiles/Dockerfile.leshan-client

```diff
@@ -0,0 +1,21 @@
+# Reference: https://github.com/rhoninl/leshan/blob/0947d3b08051b967e2cdc9908fc83c8fa89c1ee5/dockerfiles/client.dockerfile
```

Comment:
use master branch to instead commit:
https://github.com/rhoninl/leshan/blob/master/dockerfiles/client.dockerfile